test/test_texture.py

```diff
@@ -2,22 +2,25 @@
 import os
 
 import pywavefront.texture
-import pywavefront.visualization # power of two test
+import pywavefront.visualization  # power of two test
+
 
 def prepend_dir(file):
     return os.path.join(os.path.dirname(__file__), file)
 
+
 class TestTexture(unittest.TestCase):
 
     def testPathedImageName(self):
-        "For Texture objects, the image name should be the last component of the path."
+        """For Texture objects, the image name should be the last component of the path."""
         my_texture = pywavefront.texture.Texture(prepend_dir('4x4.png'))
         self.assertEqual(my_texture.image_name, prepend_dir('4x4.png'))
 
-    def testNonPowerOfTwoImage(self):
-        "Texture images that have a non-power-of-two dimension should raise an exception."
-        self.assertRaises(Exception, pywavefront.texture.Texture, prepend_dir('3x4.png'))
+    # NOTE: This us using pyglet, so disabling for now
+    # def testNonPowerOfTwoImage(self):
+    #     """Texture images that have a non-power-of-two dimension should raise an exception."""
+    #     self.assertRaises(Exception, pywavefront.texture.Texture, prepend_dir('3x4.png'))
```

Comment:
Is non-power-of-two a valid use case? Or is that just a Pyglet "thing"?

Reply:
That is a pyglet thing. They focus on having a framwork that can run on almost anything + backwards compatibility. Non power-of-two textures have been widely supported in OpenGL for 10+ years.

Reply:
The main reason for commenting out atm is that circleci can't import pyglet, so that test will fail anyway.
```
    import pywavefront.visualization # power of two test
  File "/home/circleci/repo/pywavefront/visualization.py", line 35, in <module>
    from pyglet.gl import *
ImportError: No module named pyglet.gl
```

Reply:
OK. I'd say we just remove it then. It's there in git history if we ever need it.